analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/HardcodedIpAddress.CSharp11.cs

```diff
@@ -7,7 +7,12 @@ public class HardcodedIpAddress
     {
```

Comment:
I think we should add some test cases with raw string literals also for PermissiveCors.